third_party/gtp_ovs/kernel-4.9/gtp-v4.9-backport/gtp.c

```diff
@@ -1665,7 +1705,7 @@ static int __init gtp_init(void)
 	if (err < 0)
 		goto unreg_genl_family;
 
-	pr_info("Flow-based GTP module loaded (pdp ctx size %zd bytes) : v8\n",
+	pr_info("Flow-based GTP module loaded (pdp ctx size %zd bytes) : v8h\n",
```

Comment:
can you also update version in makefile

Reply:
Comments addressed. Changed the iteration.